packages/studio-be/src/sdk/botpress.d.ts

```diff
@@ -839,6 +839,13 @@ declare module 'botpress/sdk' {
     author?: string
     disabled?: boolean
     private?: boolean
+    /**
+     * When true, the studio considers the bot as "standalone"
+     * - Auth Gate is available
+     * - We provide the runtime sdk instead of the full sdk
+     * - We add hooks create short links
+     */
+    standalone?: boolean
```

Comment:
What's the link between this and code editor ?

Reply:
The code editor loads a different SDK whether it's a cloud bot or not